### ISSUE TITLE
[bug]: Scaffolding does not work with lasted 6.0.0 release 

### ISSUE THREAD

**Issue by larsroettig** [magento/pwa-studio#2364]
**Describe the bug**
Scaffolding does not work with lasted 6.0.0 release 

**To reproduce**
1. Execute `yarn create @magento/pwa`
2. Enter project name like `pwa_test60`
3. Use sample Data Instance 

**Expected behavior**
- Scaffolding should work 

**Screenshots**
``` 
➜  WebstormProjects yarn create @magento/pwa
yarn create v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
info fsevents@1.2.11: The platform "linux" is incompatible with this module.
info "fsevents@1.2.11" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] 🔗  Linking dependencies...
warning "@magento/create-pwa > @magento/pwa-buildpack@5.1.0" has unmet peer dependency "babel-loader@~8.0.5".
warning "@magento/create-pwa > @magento/pwa-buildpack@5.1.0" has unmet peer dependency "css-loader@~2.1.1".
warning "@magento/create-pwa > @magento/pwa-buildpack@5.1.0" has unmet peer dependency "terser-webpack-plugin@~1.2.3".
warning "@magento/create-pwa > @magento/pwa-buildpack@5.1.0" has incorrect peer dependency "webpack@~4.38.0".
warning "@magento/create-pwa > @magento/pwa-buildpack@5.1.0" has unmet peer dependency "workbox-webpack-plugin@~4.2.0".
warning "@magento/create-pwa > @magento/pwa-buildpack > @magento/upward-js@4.0.1" has unmet peer dependency "express@^4.16.4".
warning "@magento/create-pwa > @magento/pwa-buildpack > graphql-playground-middleware-express@1.7.12" has unmet peer dependency "express@^4.16.2".
warning "@magento/create-pwa > @magento/pwa-buildpack > hastily@0.4.4" has unmet peer dependency "sharp@>=0.23.3".
[4/4] 🔨  Building fresh packages...
success Installed "@magento/create-pwa@1.1.0" with binaries:
      - create-pwa
@magento/create-pwa v1.1.0
Creating a PWA Studio project
? Project root directory (will be created if it does not exist) pwa_test60
? Short name of the project to put in the package.json "name" field pwa_test60
? Name of the author to put in the package.json "author" field roettigl <l.roettig@techdivision.com>
? Magento instance to use as a backend (will be added to `.env` file) 2.3.3-venia-cloud
? Braintree API token to use to communicate with your Braintree instance (will be added to `.env` file) sandbox_8yrzsvtm_s2bg8fs563crhqzk
? NPM package management client to use yarn
? Install package dependencies with yarn after creating project Yes

Running command: 

buildpack create-project pwa_test60 --template "venia-concept" --name "pwa_test60" --author "roettigl <l.roettig@techdivision.com>" --backend-url "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" --braintree-token "sandbox_8yrzsvtm_s2bg8fs563crhqzk" --npm-client "yarn"


  ℹ  Found @magento/venia-concept template in cache
  ℹ  Creating a new PWA project 'pwa_test60' in pwa_test60
  ⓧ  Error: /home/roettigl/WebstormProjects: Cannot getTargetsOf("@magento/pwa-buildpack"): @magento/pwa-buildpack has not yet declared
(node:9373) UnhandledPromiseRejectionWarning: Error: Command failed: /bin/sh -c /home/roettigl/.config/yarn/global/node_modules/@magento/pwa-buildpack/bin/buildpack create-project pwa_test60 --template "venia-concept" --name "pwa_test60" --author "roettigl <l.roettig@techdivision.com>" --backend-url "https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" --braintree-token "sandbox_8yrzsvtm_s2bg8fs563crhqzk" --npm-client "yarn"
    at makeError (/home/roettigl/.config/yarn/global/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (/home/roettigl/.config/yarn/global/node_modules/execa/index.js:278:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:9373) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:9373) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 86.97s.
```

**Additional context**

This line here is the culprit, attempting to tap into targets before we’ve installed our dependencies. Unfortunately doing DEBUG_PROJECT_CREATION couldn’t have picked up on this.
https://github.com/magento/pwa-studio/blob/v6.0.0/packages/pwa-buildpack/lib/Utilities/getEnvVarDefinitions.js#L10

**Possible solutions**
Add any ideas about possible solutions to the problem here.

**Please complete the following device information:**
 - Device [e.g. iPhone6, PC, Mac, Pixel3]:
 - OS [e.g. iOS8.1, Windows 10]:
 - Browser [e.g. Chrome, Safari]:
 - Browser Version [e.g. 22]:
 - Magento Version:
 - PWA Studio Version:
 - NPM version `npm -v`:
 - Node Version `node -v`:

**Please let us know what packages this bug is in regards to:**
- [ ] `venia-concept`
- [ ] `venia-ui`
- [ ] `pwa-buildpack`
- [ ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`
- [ ] `create-pwa`

**Comment by fooman** [comment]
Fix should hopefully be https://github.com/magento/pwa-studio/pull/2363

**Comment by larsroettig** [comment]
Fixed by #2363
